### PR TITLE
Fix video_player initialization

### PR DESCRIPTION
**Pull request by sigurdm**

### CODE REVIEW

File: packages/video_player/README.md [hunk 1]
```diff
@@ -59,29 +69,32 @@ class _MyHomePageState extends State<MyHomePage> {
           });
         }
       })
-      ..initialize();
+      ..initialize().then((_) {
+        print("Initialized ${_controller.value.initialized}");
```
Comment (mravn-google) [inline packages/video_player/README.md]: Did you intend to leave this `print` call here?

File: packages/video_player/README.md [hunk 2]
```diff
@@ -59,29 +69,32 @@ class _MyHomePageState extends State<MyHomePage> {
...
+        setState(() {});
```
Comment (mravn-google) [inline packages/video_player/README.md]: Please comment why this is needed.

File: packages/video_player/README.md (hunk 1 above)
Comment (sigurdm) [reply packages/video_player/README.md]: No!

File: packages/video_player/README.md (hunk 2 above)
Comment (sigurdm) [reply packages/video_player/README.md]: Done

**Pull request merged**